### PR TITLE
Add pygment theme

### LINKED ISSUE #30: Use better colors to highlight code on docs
Pygements demos: http://pygments.org/demo/

### PR DESCRIPTION
**Pull request by leportella**
Closes #30

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -92,7 +92,11 @@
 #
 html_theme = 'alabaster'
 html_theme_options = {
-    "pre_bg": "#61515f"
+    "pre_bg": "#49483e",
+    "github_user": "rougeth",
+    "github_repo": "bottery",
+    "github_button": True,
```
Comment (rougeth) [inline docs/conf.py]: Why do we need this button?

Comment (leportella) [reply docs/conf.py]: Thought you'd like it. removing it

Comment (rougeth) [reply docs/conf.py]: sorry :P

**Comment by rougeth** [comment]
Thank you very much @leportella!!

**Pull request merged**